app/api/routers/feature_flags.py

```
@@ -0,0 +1,163 @@
+from app.api.security.breeze_buddy.rbac_token import rbac_token_manager
+from app.core.logger import logger
+from app.schemas import UserInfo
+from app.services.live_config.store import FEATURE_FLAGS_KEY, get_all_flags
+from app.services.redis.client import get_redis_service
+
+router = APIRouter()
+security = HTTPBearer()
+
+
+async def get_current_user_with_role(
+    credentials: HTTPAuthorizationCredentials = Depends(security),
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)